package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>osqp-eigen</name>
+  <version>0.6.2</version>
```

Comment:
Again, not blocking the merge but I am tipically afraid of multiple places in which the version is defined. @GiulioRomualdi note that this may need to be updated for every new release, if necessary we can setup tooling to do this automatically.

Reply:
Otherwise we can modify the CMake to read the version from package.xml, for example dart is doing it via:
~~~
# Extract version numbers from package.xml
file(READ package.xml PACKAGE_XML)
string(REGEX MATCH "<version>[0-9]+\\.[0-9]+\\.[0-9]+</version>" DIRTY_VERSION_STRING ${PACKAGE_XML})
string(REGEX REPLACE "^<version>([0-9]+)\\.([0-9]+)\\.([0-9]+)</version>$" "\\1" DART_MAJOR_VERSION "${DIRTY_VERSION_STRING}")
string(REGEX REPLACE "^<version>([0-9]+)\\.([0-9]+)\\.([0-9]+)</version>$" "\\2" DART_MINOR_VERSION "${DIRTY_VERSION_STRING}")
string(REGEX REPLACE "^<version>([0-9]+)\\.([0-9]+)\\.([0-9]+)</version>$" "\\3" DART_PATCH_VERSION "${DIRTY_VERSION_STRING}")
set(DART_VERSION "${DART_MAJOR_VERSION}.${DART_MINOR_VERSION}.${DART_PATCH_VERSION}")
~~~
see https://github.com/dartsim/dart/blob/f07186fbdb4e89e5340abb9718d02715bb315922/CMakeLists.txt#L62 .

Reply:
Would you like me to add this to this PR or is this a comment to keep track of for future reference?

Reply:
Hi @wxmerkt, it would be nice if you could add this modification. In this case, I would also like to ask you to modify the version in the `package.xml` to [`0.6.4.100`](https://github.com/robotology/osqp-eigen/blob/974087335647bd1514bdaa08af5acebfd7723380/CMakeLists.txt#L8). We will release `0.7.0` soon

Reply:
I don't think package.xml or semver supports a third colon (beyond patch release). How about 0.6.5?

Reply:
I've made the requested updates. Note that the version convention won't allow `0.6.4.100`. It has to be major-minor-patch. Hence, I've used 0.6.4.